swift-sdk/Internal/InAppManager+Functions.swift

```diff
@@ -69,7 +69,10 @@ struct MessagesProcessor {
     }
     
     private func getFirstProcessableTriggeredMessage() -> IterableInAppMessage? {
-        messagesMap.values.filter(MessagesProcessor.isProcessableTriggeredMessage).first
+        messagesMap.values
+            .filter(MessagesProcessor.isProcessableTriggeredMessage)
+            .sorted { ($0.priorityLevel ?? Const.PriorityLevel.unassigned) < ($1.priorityLevel ?? Const.PriorityLevel.unassigned) }
```

Comment:
This logic will become easier if `priorityLevel` is not `nil`. See comments below.